reviews-receiver/src/db.ts

```diff
@@ -2,6 +2,6 @@ import { Mongoose } from 'mongoose';
 import config from './config';
 import initDB from 'revisar-server-utils/db';
 
-export const metadatadb = new Mongoose();
+export const metadata = new Mongoose();
 
-export const connectMetadatadb = async () => initDB(config.metadatadbDBUrl, config.metadatadbDBName, metadatadb);
+export const connectMetadata = async () => initDB(config.metadataDBUrl, config.metadataDBUrl, metadata);
```

Comment:
You provide the URL twice instead of the db name in the second arg

Reply:
Mehazek